### PR TITLE
F50/EventCard

### PR DESCRIPTION
**Pull request by iruseg00**

### CODE REVIEW

File: app/src/components/eventCard/EventCard.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  {
+    super(props);
+    this.state = {icon : null};
+  }
+
+  newState()
+  {
+      switch(this.props.icon)
+      {
+        case 'Аттестация':
```
Comment (i-barysiuk) [inline app/src/components/eventCard/EventCard.jsx]: not switch
maybe we may do this
const icons = [
     "Аттестация": faDraftingCompass,
]

icons["Аттестация"]

**Review by vol4k — APPROVED** [review]

File: app/src/components/eventCard/EventCard.jsx (hunk 1 above)
Comment (nikitaz1) [reply app/src/components/eventCard/EventCard.jsx]: @vania149895 , а, мы оказывается на англе комментим, ну лан

File: app/src/components/eventCard/EventCard.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  faMeteor,
+  faUmbrellaBeach,
+  faDraftingCompass,
+  faChevronRight
+} from "@fortawesome/free-solid-svg-icons";
+
+export default props=>
+{
+  const icons = {
+    'Аттестация'                    : faDraftingCompass ,
```
Comment (nikitaz1) [inline app/src/components/eventCard/EventCard.jsx:25]: You have the same file changes at another PR, that is bad practice. At future, please, wait for PR with similar changes will merge, or open PR to branch with similar changes :)

File: app/src/components/eventCard/style.module.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
+$cont-width:250px;
+$cont-height: 40px;
+$cont-radius: 7px;
+$cont-padding: 7px;
+$cont-line-height: 1.1;
+$cont-bg:#F0EEEE;
+$font-size: 11px;
```
Comment (M-Kulinkovich) [inline app/src/components/eventCard/style.module.scss:7]: plz use the variable in a separate file. see task F29
https://github.com/i-barysiuk/bstu-curators/pull/83/files

**Pull request closed without merging**